homeassistant/components/spotify/__init__.py

```diff
@@ -48,6 +48,7 @@
 
 __all__ = [
     "async_browse_media",
+    "async_get_media_browser_root_object",
```

Comment:
I don't understand why this is a new method. I would expect the root to be returned when passing `None, None` to `async_browse_media`

The cast media player would then always pass the config entry URI to browse.